dashboard/app/models/sections/section.rb

```diff
@@ -175,6 +194,20 @@ def students_attributes=(params)
     self.followers_attributes = follower_params
   end
 
+  # Checks if a user can join a section as a participant by
+  # checking if they meet the participant_type for the section
+  def can_join_section_as_participant?(user)
+    if participant_type == SharedCourseConstants::PARTICIPANT_AUDIENCE.facilitator
+      return user.permission?(UserPermission::FACILITATOR)
+    elsif participant_type == SharedCourseConstants::PARTICIPANT_AUDIENCE.teacher
+      return user.teacher?
+    elsif participant_type == SharedCourseConstants::PARTICIPANT_AUDIENCE.student
+      return true #if participant_type is student let anyone join
```

Comment:
This comment looks off because it is saying _what_ the code is doing (pretty clear in this case) instead of _why_ it is doing it. It seems like the point of this comment is to remind people that you do not actually have to exactly match the participant type in order to be eligible, is that correct? if so, I would suggest removing this comment and augmenting the method-level comment to say something like: "participant does not have to exactly match the participant type to join the section, for example facilitators can join any section and anyone can join a student section."

https://blog.codinghorror.com/code-tells-you-how-comments-tell-you-why/